src/main/java/io/turntable/projectwebservice/controllers/ProjectController.java

```diff
@@ -58,21 +62,22 @@ public void updateProjectRecord(@PathVariable String id, @RequestBody Project pr
         projectToUpdate.setProject_name(project.getProject_name());
         projectToUpdate.setDescription(project.getDescription());
         projectService.updateProject(projectToUpdate);
+        Publisher.sendMessage(String.format("project with id = %s, name = '%s' updated [UPDATE]", projectToUpdate.getProject_id(), project.getProject_name()));
     }
 
 
     @ApiOperation("get project by id")
     @GetMapping("/project/search/id/{id}")
     @ResponseStatus(HttpStatus.OK)
     public Project getProjectById(@PathVariable String id) {
+        Publisher.sendMessage(String.format("Project with id = %s [ACCESS]", id));
```

Comment:
if you change it to Integer then %s becomes %d

Reply:
yah ! yah !....i gat it

Reply:
Thanks !